recipe/meta.yaml

```diff
@@ -40,6 +44,8 @@ requirements:
 test:
   requires:
     - pip
+    # Numba needs NumPy 1.21 or less (according to numba itself)
+    - numpy <1.22.0a0
   commands:
     - pip check
   imports:
```

Comment:
We could update imports with this list:
https://github.com/slundberg/shap/blob/v0.41.0/setup.py#L221

Reply:
@cbouss `pip check` is not the problem here. The problem is that numba's code actually checks its dependencies _at runtime_ and fails.

Reply:
By import list I meant adding this:
`
- shap
- shap.explainers
- shap.explainers.other
- shap.explainers._deep
- shap.plots
- shap.plots.colors
- shap.benchmark
- shap.maskers
- shap.utils
- shap.actions
- shap.models
`

Reply:
Oh! Thank you! Will do